src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/external/SentenceCalculationType.kt

```diff
@@ -43,21 +45,18 @@ enum class SentenceCalculationType(
   SOPC18(sentenceClazz = SopcSentence::class.java),
   SOPC21(sentenceClazz = SopcSentence::class.java),
   SEC236A(sentenceClazz = SopcSentence::class.java),
-  AFINE(sentenceClazz = AFineSentence::class.java);
+  AFINE(sentenceClazz = AFineSentence::class.java, primaryName = "A/FINE");
 
   companion object {
-    fun from(sentenceCalculationType: String): SentenceCalculationType? {
-      if (sentenceCalculationType == "14FTR_ORA") {
-        return _14FTR_ORA
-      }
-      if (sentenceCalculationType == "A/FINE") {
-        return AFINE
-      }
-      return try {
-        valueOf(sentenceCalculationType)
+    fun from(sentenceCalculationType: String): SentenceCalculationType =
```

Comment:
`from` no longer returns an optional. Makes any code downstream of the validation easier (no null checks required).